modules/cloudbuild/cloudbuild_builder/Dockerfile

```diff
@@ -41,8 +43,8 @@ RUN apt-get update && \
     rm -f "${TERRAFORM_VALIDATOR_ARCHIVE}" && \
     apt-get --purge -y autoremove && \
     apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/* && \
+    chmod +x /builder/entrypoint.bash
 
 ENV PATH=/builder/terraform/:$PATH
-COPY entrypoint.bash /builder/entrypoint.bash
```

Comment:
We can  keep the copy here followed by changing the permissions next so its readable?
```docker
COPY entrypoint.bash /builder/entrypoint.bash
RUN chmod +x /builder/entrypoint.bash
```